app/gateway/2.6.x/index.md

```diff
@@ -3,81 +3,119 @@ title: Kong Gateway
+{{site.base_gateway}} is a lightweight, fast, and flexible cloud-native API
+gateway. The gateway runs in front of any RESTful API and can be extended through
+modules and plugins which provide extra functionality beyond the core platform.
 
-{{site.base_gateway}} is a Lua application running in Nginx and made possible by the [lua-nginx-module](https://github.com/openresty/lua-nginx-module). Instead of compiling Nginx with this module, {{site.base_gateway}} is distributed along with [OpenResty](https://openresty.org/), which already includes lua-nginx-module.
-<!-- OpenResty is *not* a fork of Nginx, but a bundle of modules extending its capabilities. -->
-
-This sets the foundations for a modular architecture, where Lua scripts (referred to as *plugins*) can be enabled and executed at runtime. At its core, {{site.base_gateway}} implements database abstraction, routing, and plugin management. Plugins can live in separate code bases and be injected anywhere into the request lifecycle, all with a few lines of code.
-
-{{site.base_gateway}} is designed to run on decentralized architectures, leveraging workflow automation and modern GitOps practices. With {{site.base_gateway}}, users can:
+{{site.base_gateway}} is designed to run on decentralized architectures,
+leveraging workflow automation and modern GitOps practices, and running in
+hybrid-cloud and multi-cloud deployments. With {{site.base_gateway}}, users can:
 
 * Decentralize applications/services and transition to microservices
 * Create a thriving API developer ecosystem
 * Proactively identify API-related anomalies and threats
-* Secure and govern APIs/services, and improve API visibility across the entire organization
-
-{{site.base_gateway}} is a combination of several features and modules built on top of the open-sourced {{site.base_gateway}}, as shown in the diagram and described under [{{site.base_gateway}} features](#kong-gateway-enterprise-features).
+* Secure and govern APIs/services, and improve API visibility across the
+entire organization
+
+## Modular architecture
+{{site.base_gateway}} is a Lua application running in Nginx. {{site.base_gateway}}
+is distributed along with [OpenResty](https://openresty.org/), which is a bundle
+of modules that extend the [lua-nginx-module](https://github.com/openresty/lua-nginx-module).
+
+This sets the foundations for a modular architecture, where Lua scripts
+(plugins) can be enabled and executed at runtime. At its core,
+{{site.base_gateway}} implements database abstraction, routing, and plugin
+management. Plugins can live in separate code bases and be injected anywhere
+into the request lifecycle, all with a few lines of code.
+
+## Packages and modes
+{{site.base_gateway}} is available in the following modes:
+
+**Kong Gateway (OSS)**: an open-source package containing the basic API gateway
```

Comment:
```suggestion
**Kong Gateway (OSS)**: an open-source package containing the basic API Gateway
```

Follows same question from above.

Reply:
This one is a generic term, so we shouldn't capitalize it.